cmd/assetsvc/server/server.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/heptiolabs/healthcheck"
 	"github.com/kubeapps/kubeapps/cmd/assetsvc/pkg/utils"
 	"github.com/kubeapps/kubeapps/pkg/dbutils"
-	"github.com/urfave/negroni"
+	negroni "github.com/urfave/negroni/v2"
```

Comment:
They created a tag and started using the `v2` version, but there aren't any breaking changes.
However, since snyk detects a vulnerability here, I've performed this change now so that, in the future, when dependabot creates a PR, it'll be just a matter of accepting the PR)